numpy/core/src/umath/loops.c.src

```diff
@@ -840,6 +847,62 @@ NPY_NO_EXPORT NPY_GCC_OPT_3 void
     UNARY_LOOP_FAST(@type@, @type@, *out = in > 0 ? 1 : (in < 0 ? -1 : 0));
 }
 
+#ifndef USE_LEGACY_DIVISION
+NPY_NO_EXPORT void
+@TYPE@_divide(char **args, npy_intp const *dimensions, npy_intp const *steps, void *NPY_UNUSED(func))
+{
+    static const struct libdivide_@div@_t EmptyStruct;
+    BINARY_LOOP_BASE
+
+    if(!is2) {
+        const @type@ in2 = *(@type@ *)ip2;
+        struct libdivide_@div@_t fast_d = in2 ? libdivide_@div@_gen(in2) : EmptyStruct;
```

Comment:
Is this `EmptyStruct` necessary?
```suggestion
    BINARY_LOOP_BASE

    if(!is2) {
        const @type@ in2 = *(@type@ *)ip2;
        struct libdivide_@div@_t fast_d = in2 ? libdivide_@div@_gen(in2) : {};
```

Reply:
This is better thanks, will make the change